cli.js

```diff
@@ -109,6 +111,32 @@ var cli = meow([
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+function checkDebugFlag(flag) {
+	flag = '--' + flag;
+
+	for (var i = 0; i < process.argv.length; i++) {
+		var arg = process.argv[i];
+		if (arg.indexOf(flag) === 0) {
+			arg = arg.substring(flag.length + 1) || '5858';
+			var port = parseInt(arg, 10);
+			if (isNaN(port)) {
+				console.log('Bad value for ' + flag + ' ' + arg);
+				process.exit(1);
+			}
+			return flag + '=' + port;
+		}
+	}
+
+	return null;
+}
```

Comment:
Probably not Node.

But I have ideas for `meow`: https://github.com/sindresorhus/meow/issues/44